src/components/Allocation/AllocationList.tsx

```diff
@@ -260,10 +268,11 @@ const entityToAllocation = (
   entity: AddEditAllocationModalEntity & { projectOwner: false },
   totalAllocationAmount: BigNumber | undefined,
 ): AllocationSplit => {
+  const projectIdHex = projectIdToHex(entity.projectId)
   const allocationProps = {
-    id: allocationId(entity.beneficiary!, entity.projectId),
+    id: allocationId(entity.beneficiary!, projectIdHex),
     beneficiary: entity.beneficiary,
-    projectId: entity.projectId,
+    projectId: projectIdHex,
```

Comment:
why do project ids need to be hexed?

Reply:
https://discord.com/channels/939317843059679252/1080714793595441182/1083628114179014696

See JB-93